docs/user/installation/hbase.rst

```diff
@@ -0,0 +1,145 @@
+
+In the ``geomesa-hbase_2.11-$VERSION`` directory, run ``bin/geomesa-hbase configure`` to set up the tools.
+
+.. code-block:: bash
+
+    $ bin/geomesa-hbase configure
+    Using GEOMESA_HBASE_HOME = /path/to/geomesa-hbase_2.11-1.3.0-m3-SNAPSHOT
+    Do you want to reset this? Y\n y
+    Using GEOMESA_HBASE_HOME as set: /path/to/geomesa-hbase_2.11-1.3.0-m3-SNAPSHOT
+    Is this intentional? Y\n y
+    To persist the configuration please edit conf/geomesa-env.sh or update your bashrc file to include:
+    export GEOMESA_HBASE_HOME=/path/to/geomesa-hbase_2.11-1.3.0-m3-SNAPSHOT
+    export PATH=${GEOMESA_HBASE_HOME}/bin:$PATH
+
+Update and re-source your ``~/.bashrc`` file to include the ``$GEOMESA_HBASE_HOME`` and ``$PATH`` updates.
+
+.. note::
+
+    ``geomesa-hbase`` will read the ``$HBASE_HOME`` and ``$HADOOP_HOME`` environment variables to load the
+    appropriate JAR files for Hadoop and HBase. In addition, ``geomesa-hbase`` will pull any
+    additional jars from the ``$GEOMESA_EXTRA_CLASSPATHS`` environment variable into the class path.
+    Use the ``geomesa classpath`` command in order to see what JARs are being used.
+
+Due to licensing restrictions, dependencies for shape file support and raster
+ingest must be separately installed. Do this with the following commands:
+
+.. code-block:: bash
+
+    $ bin/install-jai.sh
+    $ bin/install-jline.sh
+
+Run ``geomesa-hbase`` without arguments to confirm that the tools work.
+
+.. code::
+
+    $ bin/geomesa-hbase
+    Using GEOMESA_HBASE_HOME = /path/to/geomesa-hbase_2.11-1.3.0-m3-SNAPSHOT
+    INFO  Usage: geomesa-hbase [command] [command options]
+      Commands:
+        convert            Convert files using GeoMesa's internal converter framework
+        create-schema      Create a GeoMesa feature type
+        delete-catalog     Delete a GeoMesa catalog completely (and all features in it)
+        delete-features    Delete features from a table in GeoMesa. Does not delete any tables or schema information.
+        describe-schema    Describe the attributes of a given GeoMesa feature type
+        env                Examine the current GeoMesa environment
+        explain            Explain how a GeoMesa query will be executed
+        export             Export features from a GeoMesa data store
+        export-bin         Export features from a GeoMesa data store in a binary format
+        gen-avro-schema    Generate an Avro schema from a SimpleFeatureType
+        get-sft-config     Get the SimpleFeatureType of a feature
+        get-type-names     List GeoMesa feature types for a given catalog
+        help               Show help
+        ingest             Ingest/convert various file formats into GeoMesa
+        keywords           Add/Remove/List keywords on an existing schema
+        remove-schema      Remove a schema and associated features from a GeoMesa catalog
+        stats-analyze      Analyze statistics on a GeoMesa feature type
```

Comment:
Well, that's what you get when you run ``geomesa-hbase``! Didn't actually check each commands to see if they would do anything.